### DISCUSSION TITLE
[gatsby-plugin-emotion]: jsxRuntime automatic with gatsby-plugin-emotion i can't use @emotion/react css

### DISCUSSION THREAD

**Discussion by Ruubber** [gatsbyjs/gatsby#36833] [Help] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Hi,
I use emotion with gatsby.

I want jsxRuntime automatic but i use this config I am get bug.
I want use inline-style with css of @emotion/react.
But jsxRuntime config is automatic emotion give it to me bug.
```
<div css="You have tried to stringify object returned from `css` function. It isn't supposed to be used directly (e.g. as value of the `className` prop), but rather handed to emotion so it can handle it (e.g. as value of `css` prop)." />
```
Please Help.

I am not good at English Sorry.

### Reproduction Link

https://github.com/Ruubber/gatsby-emotion-test

### Steps to Reproduce

1. inline-style with css of @emotion/react
2. I don't want import React from "react" so i use gatsby-config jsxRuntime: "automatic"
3. emotion give it to me msg
```
<div css="You have tried to stringify object returned from `css` function. It isn't supposed to be used directly (e.g. as value of the `className` prop), but rather handed to emotion so it can handle it (e.g. as value of `css` prop)." />
```
...

### Expected Result

I think this two config come together very well.

Do I need to configure babel separately?

### Actual Result

```
<div css="You have tried to stringify object returned from `css` function. It isn't supposed to be used directly (e.g. as value of the `className` prop), but rather handed to emotion so it can handle it (e.g. as value of `css` prop)." />
```

### Environment

```shell
System:
    OS: macOS 12.4
    CPU: (8) x64 Apple M1
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.13.2 - /var/folders/_2/4ttnjsv96wxfg7x290gmqwz40000gn/T/yarn--1665966861028-0.01388999515364353/node
    Yarn: 1.22.18 - /var/folders/_2/4ttnjsv96wxfg7x290gmqwz40000gn/T/yarn--1665966861028-0.01388999515364353/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.2/bin/npm
  Browsers:
    Chrome: 106.0.5249.119
    Safari: 15.5
  npmPackages:
    gatsby: ^4.24.4 => 4.24.4 
    gatsby-plugin-emotion: ^7.24.0 => 7.24.0
```

**Comment by LekoArts** [comment, marked as answer]
As explained in https://emotion.sh/docs/css-prop#jsx-pragma and https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/#jsximportsource you'll also need to set the `jsxImportSource`
